biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.css

```diff
@@ -1,68 +1,71 @@
-#biogridVisual {
-  width: 28.125rem;
-}
-
 body {
-  background-color: #3b6d6d;
+  background-color: #0b5a5a;
 }
 
 br {
   clear: both;
 }
 
-.buildings {
+.buildings, .endDatePicker, .largeBatteryCells, 
+.smallBatteryCells, .solarPanels, .startDatePicker, 
+.townHeight, .townWidth {
   display: inline-block;
-  margin-right: 1.25rem;
-  margin-bottom: 1.25rem;
-}
+  margin: 1.25rem;
 
-.endDatePicker {
-  display: inline-block;
-  margin-top: 1.25rem;
 }
 
 .input-page {
   background-color: #f2f2f2;
-  border-radius: 0.3125rem;
+  border-radius: 1.25rem;
   font-family: sans-serif;
   margin: auto;
   padding: 1.25rem;
   text-align: center;
   width: 56.25rem;
 }
 
+#intro {
+  background: #0ca8a8;
+  border: 5px solid black;
+  border-radius: 5px;
+  color: black;
+  padding: 10px;
+}
+
 label {
   display: block;
   text-align: left;
 }
 
-.largeBatteryCells {
-  display: inline-block;
-}
-
-.smallBatteryCells {
-  display: inline-block;
-  margin-right: 1.25rem;
-  margin-bottom: 1.25rem;
-}
-
-.solarPanels {
-  display: inline-block;
+.submitButton {
+  text-align: center;
 }
 
-.startDatePicker {
+.tooltipBuild, .tooltipEndDate, .tooltipHeight, 
+.tooltipLBC, .tooltipSBC, .tooltipSolar, 
+.tooltipStartDate, .tooltipWidth {
```

Comment:
same here - create one `tooltip` class that is shared between all these tooltips